blocks/form/components/modal/modal.js

```diff
@@ -95,14 +95,19 @@ export class Modal {
   }
 }
 
-export default async function decorate(panel) {
+export default async function decorate(panel, panelJson, container, formId) {
   const modal = new Modal();
   modal.decorate(panel);
-  subscribe(panel, async (fieldDiv, formModel) => {
```

Comment:
What if we need formModel along with fieldModel, Can we get it from fieldModel or better to pass both?

Reply:
If you do `fieldModel.form` you get it. Every field model has ref to form model @jalagari